### ISSUE TITLE
Is a port for STM32L5 or STM32U5 in the roadmap? 

### ISSUE THREAD

**Issue by ronald-swiftlabs** [micropython/micropython#7280]
I am looking to port at least STM32L5. But I am seeing some challenges with regards to the secure boot and secure app. The closest thing I see is the NRF9160DK port which uses a secure boot which is also implemented/ported in MicroPython.

**Comment by dpgeorge** [comment]
Yes STM3L5 is on the roadmap, but it's not a priority and there are a lot of other things to work on first.  So you may want to look into doing this yourself.
